docs/analysis/methodology.mdx

```diff
@@ -20,32 +20,39 @@ Use higher zoom levels with caution. They yield higher-resolution results but re
 
-Others calculate the accessibility at each departure time and taking a mean of accessibility as is done by the University of Minnesota [Accessibility Observatory](http://ao.umn.edu/). Conveyal Analysis does not use this approach because the travel time to each opportunity should be treated independently. Consider a situation of a small town situated between two major cities each with 500,000 jobs, with hourly train service to each city. Suppose that we’re interested in the number of jobs reachable in one hour, given departure during a time window of 8:00 to 9:00 AM. Further suppose that, due to how the train schedules are written, it is possible to commute to all jobs in the first city in under an hour if you leave between 8:00 and 8:15, and all the jobs in the second city are accessible if you leave between 8:30 and 8:45, and at other times no jobs are accessible. This corresponds to hourly trains to each city, leaving at 8:15 in one direction and 8:45 in the other, and needing 45 minutes to travel to the respective city. Using the average of accessibility, this location would have an accessibility of 250,000, because ¼ of the time 500,000 jobs are accessible in the first city, ¼ of the time 500,000 jobs are accessible in the second city, and ½ of the time no jobs are accessible. Thus the accessibility value would be 250,000 even though there is no job that can be reached in an average travel time of less than 60 minutes.
+### Compared to mean accessibility
+Others calculate the accessibility at each departure time and taking a mean of accessibility as is done by the University of Minnesota [Accessibility Observatory](http://ao.umn.edu/). Conveyal does not use this approach because the travel time to each opportunity should be treated independently. Consider a situation of a small town situated between two major cities each with 500,000 jobs, with hourly train service to each city. Suppose that we’re interested in the number of jobs reachable in one hour, given departure during a time window of 8:00 to 9:00 AM. Further suppose that, due to how the train schedules are written, it is possible to commute to all jobs in the first city in under an hour if you leave between 8:00 and 8:15, and all the jobs in the second city are accessible if you leave between 8:30 and 8:45, and at other times no jobs are accessible. This corresponds to hourly trains to each city, leaving at 8:15 in one direction and 8:45 in the other, and needing 45 minutes to travel to the respective city. Using the average of accessibility, this location would have an accessibility of 250,000, because ¼ of the time 500,000 jobs are accessible in the first city, ¼ of the time 500,000 jobs are accessible in the second city, and ½ of the time no jobs are accessible. Thus the accessibility value would be 250,000 even though there is no job that can be reached in an average travel time of less than 60 minutes.
 
 ## Time percentile
```

Comment:
@abyrd please give the next couple subsections a look (also at https://docs-git-2023-08-updates-conveyal.vercel.app/analysis/methodology#time-percentile)